### ISSUE TITLE
Panics on windows in cmd.exe: 'Couldn't enable color support: 87'

### ISSUE THREAD

**Issue by Boscop** [bootandy/dust#70]
```
> dust
thread 'main' panicked at 'Couldn't enable color support: 87', C:\Users\me\.cargo\registry\src\github.com-1ecc6299db9ec823\du-dust-.4.5\src/main.rs:16:5
```
That's on Win 8.1. On another machine with Win 10 it works.

**Comment by bootandy** [comment]
does `dust -c` work?

**Comment by cup** [comment]
`dust -c` does not work for me

**Comment by bootandy** [comment]
above merge may fix this, we'll find out when I push a new release.

**Comment by bootandy** [comment]
@cup I shipped version v0.5.1 which should disable colors on windows 8 and therefore (hopefully) stop it crashing. Did it work? [I do not have any windows boxes]

**Comment by cup** [comment]
Its not erroring out anymore, which is great!

However its not printing this message every single time:

~~~
This version of Windows does not support ANSI colors, setting no_color flag
~~~

Probably the behavior should be:

- If Windows < 10 and `dust`, no color and print warning
- If Windows < 10 and `dust -c`, no color and no warning

**Comment by bootandy** [comment]
@Boscop Is this fixed in version v0.5.1 ?
